content/en/internal/configuration-database/service-information/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Service information"
+description: "Technical details of the Configuration Database"
+weight: 10
+type: "docs"
+---
+
+## Identity card
+
+<!-- markdownlint-disable no-inline-html no-bare-urls -->
+
+| Property                                | Value                                                                              |
+| --------------------------------------- | ---------------------------------------------------------------------------------- |
+| Name                                    | Configuration Database                                                             |
+| Description                             | Central registry of the infrastructure topology                                    |
+| URL                                     | https://goc.egi.eu                                                                 |
+| Support Email                           | `gocdb-admin` `<at>` `mailman.egi.eu`                                              |
+| [Helpdesk](../../helpdesk) Support Unit | **EGI Services and Service Components** <br/> I\_\_ Configuration Database (GOCDB) |
```

Comment:
This looks broken in the rendering

Reply:
For me https://docs.egi.eu/documentation/329/internal/configuration-database/service-information/ looks fine, what is your issue?
![Screenshot 2021-09-23 at 12 07 01](https://user-images.githubusercontent.com/451417/134489630-6f256a74-3754-4b76-a437-221b53d5f9bf.png)

Reply:
mmm, I was thinking the `|__` was a broken table cell.

So it's good

Reply:
Maybe we could find a better UTF-8 character for this. We can always change this later.